server/app/repository/HealthCheckRepository.java

```diff
@@ -0,0 +1,35 @@
+package repository;
+
+import io.ebean.DB;
+import io.ebean.Database;
+import io.ebean.SqlRow;
+import java.util.Optional;
+import javax.inject.Inject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Implements a simple query needed to check if DB is reachable */
+public final class HealthCheckRepository {
+  private final Database database;
+  private final Logger logger = LoggerFactory.getLogger(HealthCheckRepository.class);
+
+  @Inject
+  public HealthCheckRepository() {
+    this.database = DB.getDefault();
+  }
+
+  /**
+   * Does a simple query to check if the DB is reachable. In case of any exception, error is logged
+   * and an empty object is returned.
+   */
+  public Optional<SqlRow> checkDBHealth() {
```

Comment:
```suggestion
  public bool checkDBHealth() {
```

Let's simplify this so we don't expose SqlRow.